lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
```

Comment:
Hmm, ``search-tools`` is a little bit misleading as a name. Is this UI element more of a ``filter-tools`` or ``listing-filter``?